cps/kni.c

```diff
@@ -868,13 +934,18 @@ del_route(struct route_update *update, struct gk_config *gk_conf)
 void
 kni_cps_rd_event(struct cps_config *cps_conf)
 {
-	uint16_t max_route_updates = cps_conf->max_route_updates;
-	struct route_update updates[max_route_updates];
-	unsigned int i;
-	unsigned int num_updates = 0;
-	char buf[MNL_SOCKET_BUFFER_SIZE];
+	unsigned int max_rt_update_pkts = cps_conf->max_rt_update_pkts;
+	unsigned int update_pkts = 0;
```

Comment:
Initializing `update_pkts` with `cps_conf->max_rt_update_pkts` and decrementing `update_pkts` until it becomes zero eliminate the need for variable `max_rt_update_pkts`.